config/locales/models/user/en.yml

```diff
@@ -0,0 +1,73 @@
+# Structure needed for Rails i18n active record messages management
+# Example file
+en:
```

Comment:
I know, but I wanted to leave an example on how to write a model translation, a view translation and a gem translation.

Would you want me to remove this file or to just leave fields like first & last name, dni, email, and password, and remove the others?